api/src/main/java/com/redhat/ipaas/api/v1/model/Integration.java

```diff
@@ -28,6 +28,13 @@
 
     String KIND = "integration";
 
+    String LABEL = "com.redhat.ipaas/integration";
```

Comment:
Also can you document what each label represents? What's the difference between `LABEL` & `LABEL_ID` for example?